js/config.js

```diff
@@ -134,11 +134,6 @@ export const hidpi = (window.devicePixelRatio > 1 || (
     && window.matchMedia('(-webkit-min-device-pixel-ratio: 1.25),(min-resolution: 120dpi)').matches)
 );
 
-/**
- * Map configuration
- */
-export const map = _jsonMeta('map-config');
```

Comment:
You should also remove it [there](https://github.com/opendatateam/udata/blob/a8500d439f9c39d296a233183caaca48e4bbfa90/js/config.js#L187)